test/helpers/kubectl.go

```diff
@@ -401,6 +401,29 @@ func (kub *Kubectl) DaemonSetIsReady(namespace, daemonset string) (int, error) {
 	return int(d.Status.DesiredNumberScheduled), nil
 }
 
+// AddRegistryCredentials adds a registry credentials secret into the
+// cluster
+func (kub *Kubectl) AddRegistryCredentials(cred string, registry string) error {
+	if len(cred) == 0 || cred == ":" {
+		return nil
+	}
+	res := kub.ExecShort(fmt.Sprintf("%s get secret regcred", KubectlCmd))
+	if res.WasSuccessful() {
+		return nil
+	}
+	up := strings.SplitN(cred, ":", 2)
+	if len(up) != 2 {
+		return fmt.Errorf("registry credentials had an invalid format")
+	}
+
+	cmd := fmt.Sprintf("%s secret docker-registry regcred --docker-server=%s --docker-username=%s --docker-password=%s", KubectlCmd, registry, up[0], up[1])
+	res = kub.ExecShort(cmd)
+	if !res.WasSuccessful() {
```

Comment:
Nit:
```suggestion
	if !kub.ExecShort(cmd).WasSuccessful() {
```

Reply:
I don't disagree with this sentiment, but the rest of the file has the extra variable. I think we should keep the coding style consistent.